File: rc-application/src/main/java/com/transkript/reportcard/business/service/i/SequenceService.java
package com.transkript.reportcard.business.service.i;

import com.transkript.reportcard.api.dto.SequenceDto;
import com.transkript.reportcard.api.dto.response.EntityResponse;
import com.transkript.reportcard.data.entity.Sequence;
import java.util.List;

public interface SequenceService {
    
    EntityResponse create( SequenceDto var1);
    
    List<SequenceDto> getAllDto();
    
    SequenceDto getDto( Long var1);
    
    EntityResponse update( SequenceDto sequenceDto);
    
    void delete(Long id);
    
    Sequence getEntity( Long id);
}

File: rc-application/src/main/java/com/transkript/reportcard/business/service/i/SequenceService.kt
package com.transkript.reportcard.business.service.i

import com.transkript.reportcard.api.dto.SequenceDto
import com.transkript.reportcard.api.dto.response.EntityResponse
import com.transkript.reportcard.data.entity.Sequence

interface SequenceService {
    fun create(sequenceDto: SequenceDto?): EntityResponse?
    val sequences: List<SequenceDto?>?
    fun getDto(id: Long?): SequenceDto?
    fun update(id: Long?, sequenceDto: SequenceDto?): EntityResponse?
    fun delete(id: Long?): String?
    fun getEntity(sequenceId: Long?): Sequence?
}
